aiuser/response/image/generic.py

```diff
@@ -27,6 +27,9 @@ async def _post_request(self, url, payload):
             async with session.post(url=url, json=payload) as response:
                 response.raise_for_status()
                 r = await response.json()
-                image_data = base64.b64decode(r["images"][0])
-
+                if "data" in r:
+                    if "b64_json" in r["data"][0]:
+                        image_data = base64.b64decode(r["data"][0]["b64_json"])
```

Comment:
```suggestion
                if "data" in r and "b64_json" in r["data"][0]:
                    image_data = base64.b64decode(r["data"][0]["b64_json"])
```

can condense into one, just a nitpick